Content.Server/_DV/CosmicCult/Components/CosmicCultRuleComponent.cs

```diff
@@ -0,0 +1,125 @@
+using Content.Server.RoundEnd;
+using Content.Shared._DV.CosmicCult.Components;
+
+namespace Content.Server._DV.CosmicCult.Components;
+
+/// <summary>
+/// Component for the CosmicCultRuleSystem that should store gameplay info.
+/// </summary>
+[RegisterComponent, Access(typeof(CosmicCultRuleSystem))]
+public sealed partial class CosmicCultRuleComponent : Component
+{
+    /// <summary>
+    /// What happens if all of the cultists die.
+    /// </summary>
+    [DataField]
+    public RoundEndBehavior RoundEndBehavior = RoundEndBehavior.ShuttleCall;
+
+    /// <summary>
+    /// Sender for shuttle call.
+    /// </summary>
+    [DataField]
+    public string RoundEndTextSender = "comms-console-announcement-title-centcom";
+
+    /// <summary>
+    /// Text for shuttle call.
+    /// </summary>
+    [DataField]
+    public string RoundEndTextShuttleCall = "cosmiccult-elimination-shuttle-call";
+
+    /// <summary>
+    /// Text for announcement.
+    /// </summary>
+    [DataField]
+    public string RoundEndTextAnnouncement = "cosmiccult-elimination-announcement";
+
+    /// <summary>
+    /// Time for emergency shuttle arrival.
+    /// </summary>
+    [DataField]
+    public TimeSpan EvacShuttleTime = TimeSpan.FromMinutes(5);
+
+    [DataField]
+    public HashSet<EntityUid> Cultists = new();
```

Comment:
VSCode annoys me about this :^)

```suggestion
    public HashSet<EntityUid> Cultists = [];
```